libs/portals/admin/ids-admin/src/lib/messages.ts

```diff
@@ -778,4 +778,8 @@ export const m = defineMessages({
     defaultMessage:
       'Should personal representatives automatically get this scope for their clients',
   },
+  clientIdAlreadyExists: {
+    id: 'ap.ids-admin:client-id-already-exists',
+    defaultMessage: 'ClientId already exists',
```

Comment:
Just to match name on label

```suggestion
    defaultMessage: 'Client ID already exists',
```